honeycombio/type_helpers.go

```diff
@@ -156,10 +156,10 @@ func tpmToFloat(t int) float64 {
 }
 
 func flattenNotificationRecipients(rs []honeycombio.NotificationRecipient) []map[string]interface{} {
-	result := make([]map[string]interface{}, len(rs))
+	result := []map[string]interface{}{}
```

Comment:
Why did you change this slice construction? The `make` that was here is more idiomatic, IMO.

Reply:
Ah, fair. I overcorrected for inserting by index (instead of append). I can flip this back 👍🏻